nss-day-cohort-52/rare-client-smoky-pythons
# PostList component is now linked to the /posts route from the navbar

…ositories directory with a posts repository module

### Issue ticket(s): #24 #18

### New file(s): Post.js, PostList.js, PostRepository.js, Settings.js

### Modified File(s): ApplicationViews.js, NavBar.js, 

### Modifications: Made it so clicking on the posts link in the nav bar now takes you to /posts route and renders the PostList component

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to al-viewPosts
3. login to the application
4. click on the "posts" link in the nav bar...
5. the url should now have a "/posts" extension on it and the PostList component should be rendered (just renders a sentence right now)
